tools/cldr-code/src/main/resources/org/unicode/cldr/draft/keyboard/osx/osx-locales.csv

```diff
@@ -85,9 +85,9 @@ Maltese,mt,
 Maori,mi,
 Nepali,ne,
 Northern Sami,se,
-Norwegian,no,
-Norwegian Extended,no,extended
-NorwegianSami-PC,no,extended-var
+Norwegian,nb,
+Norwegian Extended,nb,extended
+NorwegianSami-PC,nb,extended-var
```

Comment:
i don't know how the keyboard files are used, but seems plausible

Reply:
I was assuming that what was being referred to here was the locale ID used on the platform. But this actually goes back to the code that was used in CLDR 38, I had chnaged in my previous Norwegian PR since nb was no longer valid.